apps/web/src/screens/Profile/components/TransactionHistory/components/TransactionStatus.tsx

```diff
@@ -0,0 +1,59 @@
+import type { FC } from 'react';
+import { StyleSheet } from 'react-native';
+import { Text, View } from '@walless/gui';
+import { Check, Clock, Times } from '@walless/icons';
+
+interface Props {
+	status?: 'success' | 'pending' | 'failed';
+}
+
+const TransactionStatus: FC<Props> = ({ status = 'success' }) => {
+	const capitalize = (text: string) =>
+		text.charAt(0).toUpperCase() + text.slice(1);
```

Comment:
make statuses capitalized instead of making change below, these statuses are just used in this component

Reply:
i don't get what you mean

Reply:
status?: 'Success' | 'Pending' | 'Failed'; and just reuse them